### DISCUSSION TITLE
Two different words are getting merged while extract text from PDF

### DISCUSSION THREAD

**Discussion by sarveshkrg** [jsvine/pdfplumber#800] [Ask for help with specific PDFs]
Two different words are getting merged while extracting text from PDF.

Example
Actual text is **'the full'** but while reading it using extract_text method we are getting **'thefull'**.

Code.
`import pdfplumber`
`with pdfplumber.open("stayveer-epar-product-information_en.pdf") as pdf:`
`    page = pdf.pages[1]`
`    print(page.extract_text())`

PDF screenshot
![image](https://user-images.githubusercontent.com/46168717/216142877-4e02c423-c6cd-4339-9040-2052a4cc6ae9.png)

PDF file used
[stayveer-epar-product-information_en.pdf](https://www.ema.europa.eu/en/documents/product-information/stayveer-epar-product-information_en.pdf)

**Comment by jsvine** [comment]
Hi @sarveshkrg, and thanks for your interest in this library. Have you tried adjusting the `.extract_text(x_tolerance=...)` parameter? See [this part of the documentation](https://github.com/jsvine/pdfplumber#the-pdfplumberpage-class).
